src/scss/modules/table-list.scss

```diff
@@ -39,7 +39,7 @@
 
 .table-list__cell {
   overflow: hidden;
-  padding: 16px;
+  padding: 16px 8px;
```

Comment:
I went with `8px` instead of `4px` here for the left/right padding. `8px` seems like enough to accommodate the updated/added date with the full year.